src/main/java/edu/greenblitz/tobyDetermined/Nodesssss/NodeBase.java

```diff
@@ -0,0 +1,152 @@
+
+import static edu.greenblitz.tobyDetermined.RobotMap.TelescopicArm.Elbow.STARTING_ANGLE_RELATIVE_TO_GROUND;
+
+public class NodeBase  {
+    private NodeArm REST_ABOVE_BELLY;
+    private NodeArm INTAKE_GRAB_CUBE_POSITION;
+    private NodeArm CONE_HIGH;
+    private NodeArm CONE_MID;
+    private NodeArm CUBE_HIGH;
+    private NodeArm CUBE_MID;
+    private NodeArm LOW;
+    private NodeArm ZIG_HAIL;
+    private NodeArm INTAKE_GRAB_CONE_POSITION;
+    private NodeArm PRE_CONE_DROP;
+    private NodeArm POST_CONE_DROP;
+    private NodeArm currentNode;
+    private int i = 1;
+    private final LinkedList<NodeArm> list = new LinkedList<>();
+    private static NodeBase instance;
+    private final double TOLERANCE_ANGLE = Units.degreesToRadians(3);
+
+    private final double TOLERANCE_LENGTH = 0.04;//In Meters
+
+    public static NodeBase getInstance() {
+        init();
+        return instance;
+    }
+
+    public static void init() {
+        if (instance == null) {
+            instance = new NodeBase();
+        }
+    }
+
+    public int addToList(NodeArm nodeArm, LinkedList<NodeArm> list){
+        list.add(i,nodeArm);
+        i=i+1;
+        return i;
+    }
+    public NodeBase(){
+        /*
+        double[][] poses = {{1,5},{2,6}};
+
+      for (int j = 0; j < poses.length; j++) {
+            addToList(new NodeArm(j, poses[j][0],poses[j][1]), list);
+
+            list.get(0).setNeighbors();
+        }*///todo change list adding to this format
+
+        list.add(0,null);
+        i = addToList(CONE_HIGH = new NodeArm( i, 0.71, Math.toRadians(25.1) - STARTING_ANGLE_RELATIVE_TO_GROUND), list);
+        i = addToList(CONE_MID = new NodeArm( i, 0.31, /*1.94*/ Math.toRadians(107)),list);
+        i = addToList(CUBE_HIGH = new NodeArm( i, 0.450, Math.toRadians(15.46) - STARTING_ANGLE_RELATIVE_TO_GROUND),list);
+        i = addToList(CUBE_MID = new NodeArm( i, 0.29, 1.85),list);
+        i = addToList(LOW = new NodeArm( i, 0.35, Math.toRadians(60)),list);
+        i = addToList(ZIG_HAIL = new NodeArm( i, 0, Math.toRadians(20.7) - STARTING_ANGLE_RELATIVE_TO_GROUND),list);
+        i = addToList(INTAKE_GRAB_CONE_POSITION = new NodeArm( i, 0.34, 0.123),list);
+        i = addToList(INTAKE_GRAB_CUBE_POSITION = new NodeArm(i,0.25, 0.123),list);
+        i = addToList(REST_ABOVE_BELLY = new NodeArm(i,-0.02,0.196),list);
+        i = addToList(PRE_CONE_DROP = new NodeArm(i,0.089,0.667),list);
+        i = addToList(POST_CONE_DROP = new NodeArm(i,0.080,0.1),list);
+
+        CONE_HIGH.setNeighbors(new NodeArm[] {CONE_MID, CUBE_HIGH, CUBE_MID, ZIG_HAIL,PRE_CONE_DROP});
+        CONE_MID.setNeighbors(new NodeArm[] {CONE_HIGH, CUBE_HIGH,  CUBE_MID, ZIG_HAIL,PRE_CONE_DROP});
+        CUBE_HIGH.setNeighbors(new NodeArm[] {CONE_MID, CONE_HIGH,  CUBE_MID, ZIG_HAIL,PRE_CONE_DROP});
+        CUBE_MID.setNeighbors(new NodeArm[] {CONE_MID, CUBE_HIGH,  CONE_HIGH, ZIG_HAIL,PRE_CONE_DROP});
+        LOW.setNeighbors(new NodeArm[] {ZIG_HAIL});
```

Comment:
why is low not connected to the others?

Reply:
because than it is colliding with the rotating belly